f5/bigip/resource.py

```diff
@@ -369,6 +377,48 @@ def raw(self):
         """
         return self.__dict__
 
+    def _key_dot_replace(self, rdict):
```

Comment:
Why is this necessary?   Can we add a test that fails without the presence of this helper?